app/uploaders/profile_image_uploader.rb

```diff
@@ -14,6 +14,10 @@ def extension_whitelist
     %w[jpg jpeg jpe gif png ico bmp dng]
   end
 
+  def size_range
+    1..2.megabytes
```

Comment:
How does this work exactly? I'd think `1..2` would mean that I couldn't upload anything _less than_ 1 mb, or am I missing something?

Reply:
The range is from 1 byte to 2 megabytes. I read about it [here](https://github.com/carrierwaveuploader/carrierwave/wiki/How-to%3A-Validate-image-file-size).